packages/core/content-type-builder/admin/src/components/IconPicker/index.js

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useRef } from 'react';
+import {
+  Box,
+  Flex,
+  Icon,
+  Typography,
+  Searchbar,
+  IconButton,
+  inputFocusStyle,
+  VisuallyHidden,
+  Field,
+  FieldLabel,
+  FieldInput,
+} from '@strapi/design-system';
+import * as Icons from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { getTrad } from '../../utils';
+
+const EXCLUDE_ICONS = [
```

Comment:
Just curious why you decide to go for a blacklist approach and not a whitelist

Reply:
My concern is that if people add new icons to the DS, they probably won't remember to add them to this blacklist, so we could end up with icons that look out of place (like the colored ST/CT icons before)

I would personally more see a whitelist approach since I assume the list of icons we do want here is pretty much definite. I'd like to hear what others think about this though @markkaylor @gu-stav

Reply:
I think this thread is no longer valid, because @Feranchz made it an explicitly allowlist instead of a blocklist.